[2024-11-21T11:40:26.423+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-21T11:40:26.452+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.transform_cleaning manual__2024-11-21T11:35:14.427550+00:00 [queued]>
[2024-11-21T11:40:26.470+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.transform_cleaning manual__2024-11-21T11:35:14.427550+00:00 [queued]>
[2024-11-21T11:40:26.472+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-21T11:40:26.517+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_cleaning> on 2024-11-21 11:35:14.427550+00:00
[2024-11-21T11:40:26.531+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3975) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-21T11:40:26.536+0000] {standard_task_runner.py:72} INFO - Started process 3984 to run task
[2024-11-21T11:40:26.536+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'transform_cleaning', 'manual__2024-11-21T11:35:14.427550+00:00', '--job-id', '241', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp10mdfkcr']
[2024-11-21T11:40:26.538+0000] {standard_task_runner.py:105} INFO - Job 241: Subtask transform_cleaning
[2024-11-21T11:40:26.619+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.transform_cleaning manual__2024-11-21T11:35:14.427550+00:00 [running]> on host 7a40aa914b11
[2024-11-21T11:40:27.071+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_cleaning' AIRFLOW_CTX_EXECUTION_DATE='2024-11-21T11:35:14.427550+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-21T11:35:14.427550+00:00'
[2024-11-21T11:40:27.075+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-21T11:40:27.077+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-21T11:40:27.080+0000] {logging_mixin.py:190} INFO - Current task name:transform_cleaning state:running start_date:2024-11-21 11:40:26.453430+00:00
[2024-11-21T11:40:27.083+0000] {logging_mixin.py:190} INFO - Dag name:etl_pipeline and current dag run status:running
[2024-11-21T11:40:27.084+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-21T11:40:27.086+0000] {logging_mixin.py:190} INFO - Memproses file 'Merged_Sunscreen Azarine.csv'...
[2024-11-21T11:40:27.137+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:48: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.loc[:, 'periode_penggunaan'] = df['usage_period'].map(usage_period_mapping_id)

[2024-11-21T11:40:27.140+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:51: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.rename(columns={'rating_count': 'rating'}, inplace=True)

[2024-11-21T11:40:27.141+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:52: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.rename(columns={'value': 'trend'}, inplace=True)

[2024-11-21T11:40:27.145+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:55: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[['skintype', 'skintone', 'undertone']] = df['profile_description'].str.split(',', expand=True)

[2024-11-21T11:40:27.148+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:55: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[['skintype', 'skintone', 'undertone']] = df['profile_description'].str.split(',', expand=True)

[2024-11-21T11:40:27.150+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:55: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[['skintype', 'skintone', 'undertone']] = df['profile_description'].str.split(',', expand=True)

[2024-11-21T11:40:27.159+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:61: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.drop(columns=['usage_period'], inplace=True)

[2024-11-21T11:40:27.164+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:62: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.drop(columns=['age'], inplace=True)

[2024-11-21T11:40:27.168+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:63: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.drop(columns=['recommend'], inplace=True)

[2024-11-21T11:40:27.171+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:64: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.drop(columns=['profile_description'], inplace=True)

[2024-11-21T11:40:27.203+0000] {logging_mixin.py:190} INFO - Data telah disimpan di: dags/hasil data transform/Clean_Sunscreen Azarine.csv
[2024-11-21T11:40:27.205+0000] {logging_mixin.py:190} INFO - Memproses file 'Merged_Sunscreen Skinaqua.csv'...
[2024-11-21T11:40:27.226+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:48: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.loc[:, 'periode_penggunaan'] = df['usage_period'].map(usage_period_mapping_id)

[2024-11-21T11:40:27.228+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:51: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.rename(columns={'rating_count': 'rating'}, inplace=True)

[2024-11-21T11:40:27.229+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:52: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.rename(columns={'value': 'trend'}, inplace=True)

[2024-11-21T11:40:27.231+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:55: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[['skintype', 'skintone', 'undertone']] = df['profile_description'].str.split(',', expand=True)

[2024-11-21T11:40:27.232+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:55: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[['skintype', 'skintone', 'undertone']] = df['profile_description'].str.split(',', expand=True)

[2024-11-21T11:40:27.233+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:55: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[['skintype', 'skintone', 'undertone']] = df['profile_description'].str.split(',', expand=True)

[2024-11-21T11:40:27.239+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:61: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.drop(columns=['usage_period'], inplace=True)

[2024-11-21T11:40:27.241+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:62: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.drop(columns=['age'], inplace=True)

[2024-11-21T11:40:27.243+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:63: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.drop(columns=['recommend'], inplace=True)

[2024-11-21T11:40:27.244+0000] {warnings.py:112} WARNING - /opt/***/dags/etl/transform_cleaning.py:64: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df.drop(columns=['profile_description'], inplace=True)

[2024-11-21T11:40:27.261+0000] {logging_mixin.py:190} INFO - Data telah disimpan di: dags/hasil data transform/Clean_Sunscreen Skinaqua.csv
[2024-11-21T11:40:27.263+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-21T11:40:27.276+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-21T11:40:27.278+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=transform_cleaning, run_id=manual__2024-11-21T11:35:14.427550+00:00, execution_date=20241121T113514, start_date=20241121T114026, end_date=20241121T114027
[2024-11-21T11:40:27.293+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-21T11:40:27.294+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-21T11:40:27.295+0000] {logging_mixin.py:190} INFO - Dag name:etl_pipeline queued_at:2024-11-21 11:35:14.595004+00:00
[2024-11-21T11:40:27.296+0000] {logging_mixin.py:190} INFO - Task hostname:7a40aa914b11 operator:PythonOperator
[2024-11-21T11:40:27.321+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-21T11:40:27.374+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-21T11:40:27.382+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
